.circleci/config.yml

```diff
@@ -95,6 +95,7 @@ jobs:
     parallelism: 10
     environment:
       COVERALLS_PARALLEL: true
+      HYRAX_VALKYRIE: true
```

Comment:
Is this creating another test run that will run all the tests again with a config stating that use_valkyrie=true?

Reply:
this turns on valkyrie use for all test runs.

we may want to consider adding this to the matrix. if so, which builds do we need to duplicate?

Reply:
It seems like, until we have removed all references to AF and tests are updated to create and pass valkyrie objects, then we are still supporting AF in our code and need the tests to run with AF and again with Valkyrie.

For file_set_actor_spec, I run all the tests with 
```
[true, false].each do |use_valkyrie|
```

I would think something similar here would work.

Reply:
Lets move to running the full matrix with `HYRAX_VALKYRIE=0` and just the Ruby 2.6/Rails 5.2 build with `HYRAX_VALKYRIE=1` for now.

As we get closer to native-valkyrie throughout, we can flip the order.

`[true, false].each do |use_valkyrie|` won't work here, since the actor initializer and `#create` method signatures are defined elsewhere. There's no nice way to inject this.

Besides, I'm keen to have an environment variable so folks can start testing the more experimental valkyrie-native work in real applications.